### PR TITLE
 Add new templeate for development issue

### PR DESCRIPTION
**Pull request by maxbbn**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/development-issue.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+name: Development Issue
+about: 关于技能开发， 语音接入， 智能家居接入， 接入协议相关的问题
```
Comment (yorkie) [inline .github/ISSUE_TEMPLATE/development-issue.md]:
```suggestion
about: 关于技能开发，语音接入，智能家居接入，接入协议相关的问题
```

File: .github/ISSUE_TEMPLATE/development-issue.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+---
+name: Development Issue
```
Comment (yorkie) [inline .github/ISSUE_TEMPLATE/development-issue.md]: What's the "development issue"?

File: .github/ISSUE_TEMPLATE/development-issue.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+about: 关于技能开发， 语音接入， 智能家居接入， 接入协议相关的问题
+title: ''
+labels: development
```
Comment (yorkie) [inline .github/ISSUE_TEMPLATE/development-issue.md]: development are more generic, we should describe in modules.

File: .github/ISSUE_TEMPLATE/development-issue.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+assignees: yorkie
+
+---
+
+**请详细描述你的问题**
+
+请描述你的问题， 前置条件
```
Comment (yorkie) [inline .github/ISSUE_TEMPLATE/development-issue.md]: The steps to reproduce should be required for issues.

File: .github/ISSUE_TEMPLATE/development-issue.md (hunk 2 above)
Comment (maxbbn) [reply .github/ISSUE_TEMPLATE/development-issue.md]: How about add a title  "Ask a question" .  Guys may be confused by something, thay can come here to ask for it

File: .github/ISSUE_TEMPLATE/development-issue.md (hunk 3 above)
Comment (maxbbn) [reply .github/ISSUE_TEMPLATE/development-issue.md]: yes， you are right, see the previous suggestion

File: .github/ISSUE_TEMPLATE/development-issue.md (hunk 2 above)
Comment (yorkie) [reply .github/ISSUE_TEMPLATE/development-issue.md]: Ask a question LGTM

File: .github/ISSUE_TEMPLATE/development-issue.md (hunk 3 above)
Comment (yorkie) [reply .github/ISSUE_TEMPLATE/development-issue.md]: After changing to "ask a question", the label should be "question".

**Comment by maxbbn** [comment]
@yorkie 没有问题就合并掉吧

**Review by yorkie — APPROVED** [review]

**Pull request merged**